packages/cx-elements/src/components/Element/typography.token.ts

```diff
@@ -1,5 +1,54 @@
-import { asTokenCollection } from '../../util';
+import { as } from '@bodiless/fclasses';
+import { asTokenGroup } from '../../util';
 
-export const cxElementTypography = asTokenCollection({
+const meta = {
+  categories: {
+    Type: ['Element'],
+    Group: ['Typography'],
+  },
+};
+
+// TODO/DISCUSS
+// I brought in color to typography but should typography + color = another token ????
+// I tried first cxelementcolor but that wasn't working so for now its actual class
+// import { cxElementColor } from './color.token';
+
+export const cxElementTypography = asTokenGroup(meta)({
```

Comment:
@wodenx  Should typography include the color? 
if so is there better way to bring cxElementColor rather than applying the class directly -- the commented out code wasn't being picked up.
or compose a new token that includes typography size + & color?